.gitignore

```diff
@@ -42,3 +42,16 @@ errors.log
 /TPL/libsvm-3.1-custom/svm-prob
 /TPL/libsvm-3.1-custom/svm-scale
 /TPL/libsvm-3.1-custom/svm-train
+
+.DS_Store
+.idea/
+bin/CreateIngest.py
```

Comment:
These look like old files from before a rename commit a while ago. Mac is silly about casing (forces case insensitive options), so unless you have active changes in the bin directory, probable blow away the bin directory and check it back it. To confirm that things are in the correct state after you blow it away and check it out again, casing should follow camel case rules with the first letter being lower case. `run_SMQTK_Application.py` should just be `runApplication.py`.